vhdbuilder/packer/backfill-cleanup.sh

```diff
@@ -53,15 +53,15 @@ if [[ -n "${AZURE_RESOURCE_GROUP_NAME}" ]]; then
     image_defs=$(az sig image-definition list -g ${AZURE_RESOURCE_GROUP_NAME} -r ${gallery} | jq -r '.[] | select(.osType == "Windows").name')
     for image_definition in $image_defs; do
         echo "Finding sig image versions associated with ${image_definition} in gallery ${gallery}"
-        image_versions=$(az sig image-version list -g ${AZURE_RESOURCE_GROUP_NAME} -r ${gallery} -i ${image_definition} | jq --arg dl $deadline -r '.[] | select(.tags.now < $dl).name')
-        for image_version in $image_versions; do
-            az sig image-version show -e $image_version -i ${image_definition} -r ${gallery} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id
-            echo "Deleting sig image-version ${image_version} ${image_definition} from gallery ${gallery} rg ${AZURE_RESOURCE_GROUP_NAME}"
-            az sig image-version delete -e $image_version -i ${image_definition} -r ${gallery} -g ${AZURE_RESOURCE_GROUP_NAME}
+        old_image_versions=$(az sig image-version list -g ${AZURE_RESOURCE_GROUP_NAME} -r ${gallery} -i ${image_definition} | jq --arg dl $deadline -r '.[] | select(.tags.now < $dl).name')
+        for old_image_version in $old_image_versions; do
+            az sig image-version show -e $old_image_version -i ${image_definition} -r ${gallery} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id
```

Comment:
unrelated, but do we really need this call to `az sig image-version show`? seems like this would slow things down a fair amount